mods/gen4/formats-data.js

```diff
@@ -1099,7 +1099,7 @@ exports.BattleFormatsData = {
 	},
 	tyranitar: {
 		inherit: true,
-		randomBattleMoves: ["crunch", "stoneedge", "pursuit", "superpower", "earthquake", "dragondance", "icepunch", "fireblast", "icebeam"],
+		randomBattleMoves: ["crunch", "stoneedge", "pursuit", "superpower", "earthquake", "dragondance", "icepunch", "firepunch", "stealthrock"],
```

Comment:
Why? You do know Randoms has 84 EVs in everything

Reply:
Because the physical moves fit with DD, and Tyranitar almost always seems to roll DD in my experience (probably because it has a big movepool and everything except Crunch/Stone Edge/DD gets rejected until it has a STAB). The physical moves also increase the chance that it could roll CB, which is a better set than Expert Belt imo. Also, TTar's Ice Punch is stronger than Ice Beam against equal defenses, even with the balanced EVs.

That's my reasoning, but I'll add Fire Blast and Ice Beam back if you still disagree.